packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -473,14 +473,13 @@ export const buildStyledComponent = (tag: Tag, cssOutput: CSSOutput, meta: Metad
   // Rely on transformCss to remove duplicates and return only the last unconditional CSS for each property
   const uniqueUnconditionalCssOutput = transformCss(unconditionalCss.join(''));
 
-  // Rely on transformItemCss to build logicalExpressions for logical CSS
-  const logicalCssOutput = transformItemCss({ css: logicalCss, variables: cssOutput.variables });
-
-  const sheets = [...uniqueUnconditionalCssOutput.sheets, ...logicalCssOutput.sheets];
+  // Rely on transformItemCss to build expressions for conditional & logical CSS
+  const conditionalCssOutput = transformItemCss(conditionalCss);
```

Comment:
Oh so `transformItemCss` never did anything with `variables: cssOutput.variables`?

Reply:
Apparently not. I double checked several times to make sure I wasn't going crazy 😂 haha